### DISCUSSION TITLE
Dyness battery supported?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1381] [Q&A] [answered]
Hey guys, 
is Dyness (and conecutively Dyness BMS) supported by OpenDTU-OnBattery?
Example https://suministrosdelsol.com/en/dyness-batteries/892-dyness-b4850-48v-24kwh-lithium-battery.html

**Comment by AndreasBoehm** [comment]
Hey, the battery is compatible with victron inverters and our support for pytes batteries is also based on the victron protocol. Because of that i would assume that the battery can work with OpenDTU-OnBattery, but there is no guarantee until someone tried it or provides a CAN Protocol documentation from dyness.

**Reply by greymda** [reply]
if found this on internet, but don't know where exactly to look https://drive.google.com/file/d/1hSka8UvoSMR2WizIR3u58ZHuIAa3Q9Xh/view

**Reply by greymda** [reply]
and this https://solar.co.za/wp-content/uploads/2019/09/Dyness-Victron-inverter-Setup-startshut-down.pdf

**Comment by AndreasBoehm** [comment, marked as answer]
Based on this issue i assume that Dyness works exactly the same as a Pylontech battery: https://github.com/victronenergy/venus/issues/913

**Reply by greymda** [reply]
thanks a lot, @AndreasBoehm!

**Reply by hokreb** [reply]
Did you try it already out with the dyness batteries? I have a system with 4 dyness b4850 and would also like to try this out, so any feedback is appriciated

**Reply by Charly-2** [reply]
Hi ! 
I use 2 DYNESS DL5.0c over CAN-Bus and it works without problems!
<img width="1620" height="1115" alt="2025-07-14 13_14_37-OpenDTU-OnBattery" src="https://github.com/user-attachments/assets/6f2355de-17d5-4628-804d-d5445876892d" />
